branchLayout/impl/src/main/java/com/virtuslab/branchlayout/impl/readwrite/BranchLayoutFileWriter.java

```diff
@@ -16,13 +16,15 @@
 import com.virtuslab.branchlayout.api.IBranchLayout;
 import com.virtuslab.branchlayout.api.IBranchLayoutEntry;
 import com.virtuslab.branchlayout.api.readwrite.IBranchLayoutWriter;
+import com.virtuslab.qual.guieffect.UIThreadUnsafe;
 
 @ExtensionMethod(BranchLayoutFileUtils.class)
 @CustomLog
 @RequiredArgsConstructor
 public class BranchLayoutFileWriter implements IBranchLayoutWriter {
 
   // TODO (#692): wrap write operation in WriteAction
```

Comment:
Unrel... @mkondratek can it be removed? as #692 is closed 🤔

Reply:
yes, I think we can remvoe it